docs/BackToTopButton.md

```diff
@@ -0,0 +1,68 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Go Top Button
+
+`BackToTopButton` is a store block that redirects users to the top of the page when clicked on. This Component can be imported and used by any VTEX app.
+
+![image](https://user-images.githubusercontent.com/28419764/77644893-9238af80-6f40-11ea-8ceb-7355d0c12686.png)
+
+Icon Style:
```

Comment:
```suggestion
```

Reply:
@guerreirobeatriz Thanks for the suggestions. It's done.